docker-compose.yml

```diff
@@ -1,171 +1,182 @@
+      - /var/log/multi-cloud:/var/log/multi-cloud
+      - ./examples/multi-cloud.conf:/etc/multi-cloud/multi-cloud.conf
+  
+  file:
+    image: opensdsio/multi-cloud-file
+    environment:
+      - MICRO_REGISTRY=mdns
+      - DB_HOST=datastore:27017
+    volumes:
+      - /etc/localtime:/etc/localtime
+      - /var/log/multi-cloud:/var/log/multi-cloud
+      - ./examples/multi-cloud.conf:/etc/multi-cloud/multi-cloud.conf
+
+  s3:
+    image: opensdsio/multi-cloud-s3
+    environment:
+      - MICRO_REGISTRY=mdns
+      #- DB_HOST=datastore:27017
+      - DB_HOST=tidb:4000
+      # 0 means use user defined storage class, otherwise use default storage class.
+      - USE_DEFAULT_STORAGE_CLASS=1
+    volumes:
+      - /etc/localtime:/etc/localtime
+      - /var/log/multi-cloud:/var/log/multi-cloud
+      - ./s3/pkg/conf:/etc/multi-cloud
+      - ./s3/pkg/datastore/yig/conf/:/etc/yig
+      - /etc/ceph:/etc/ceph
+      - ./examples/multi-cloud.conf:/etc/multi-cloud/multi-cloud.conf
+      - ./s3/pkg/conf/tidb.sql:/s3.sql
+      - ./s3/pkg/datastore/yig/conf/yig.sql:/yig.sql
+    depends_on:
+      - redis
+      - tidb
+
+  dataflow:
+    image: opensdsio/multi-cloud-dataflow
+    environment:
+      - MICRO_REGISTRY=mdns
+      - DB_HOST=datastore:27017
+      - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092
+      - LIFECYCLE_CRON_CONFIG=0 0 0 * * ?
+    volumes:
+      - /etc/localtime:/etc/localtime
+      - /var/log/multi-cloud:/var/log/multi-cloud
+      - ./examples/multi-cloud.conf:/etc/multi-cloud/multi-cloud.conf
+    depends_on:
+      - zookeeper
+      - kafka
+      - s3
+
+  datamover:
+    image: opensdsio/multi-cloud-datamover
+    volumes:
+      - /etc/ssl/certs:/etc/ssl/certs
+      - /etc/localtime:/etc/localtime
+      - /var/log/multi-cloud:/var/log/multi-cloud
+      - ./examples/multi-cloud.conf:/etc/multi-cloud/multi-cloud.conf
+    environment:
+      - MICRO_REGISTRY=mdns
+      - DB_HOST=datastore:27017
+      - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092
+      # PARTSIZE for multi-part migration, should not be less than 5(MB) or more than 100(MB)
+      - PARTSIZE=5
+      # JOB_MAX_RUN_TIME for each migration job, should not be less than 60(seconds) or more than 2592000(seconds)
+      - JOB_MAX_RUN_TIME=86400
+      # For lifecycle, if object need to be moved from one place to another, how long this moving can continue, should
+      # not be less than 60(seconds) or more than 2592000(seconds).
+      - OBJECT_MOVE_TIME=3600
+    depends_on:
+      - zookeeper
+      - kafka
+
+  datastore:
+    image: mongo
+    ports:
+      - 27017:27017
+    volumes:
+      - ./mongodb_data:/data/db
+
+  tidb:
+    image: pingcap/tidb:v2.1.16
+    container_name: tidb
+    ports:
+      - "4000:4000"
+      - "10080:10080"
+    volumes:
+      - ./s3/tidbconf/tidb.toml:/tidb.toml:ro
+      - ./tidb_data:/tmp/tidb
+    command:
+      - --store=mocktikv
+      - --config=/tidb.toml
+      - -L=info
```

Comment:
Sorry, why changed all exist configurations? Do we just need to add a few lines?

Reply:
BTW, suggest to seperate vendor changes from others.